src/dashboard/Data/Browser/ExportSelectedRowsDialog.react.js

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2016-present, Parse, LLC
+ * All rights reserved.
+ *
+ * This source code is licensed under the license found in the LICENSE file in
+ * the root directory of this source tree.
+ */
+import Modal     from 'components/Modal/Modal.react';
+import React     from 'react';
+
+export default class ExportSelectedRowsDialog extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      confirmation: ''
+    };
+  }
+
+  valid() {
+    return true;
+  }
+
+  render() {
+    let selectionLength = Object.keys(this.props.selection).length;
+    return (
+      <Modal
+        type={Modal.Types.INFO}
+        icon='warn-outline'
+        title={this.props.selection['*'] ? 'Export all rows?' : (selectionLength === 1 ? 'Export this row?' : `Export ${selectionLength} rows?`)}
```

Comment:
Can you change this to `Export selected row` instead of `Export this row`? Looking at the menu, it is not clear what `this / these` is referring to.

Reply:
done

Reply:
I agree with the change, though I pulled the wording from the other options in the Edit menu. Perhaps this is a change that should be made (in another PR) for the other options in Edit?